### DISCUSSION TITLE
Good startup guide for newbies (like me)

### DISCUSSION THREAD

**Discussion by raribeir** [oracle/docker-images#2076] [General]
I recently needed to get Oracle in an Ubuntu VM for specific purposes and I found https://mkc110891.medium.com/oracle-19c-on-ubuntu-18-04-docker-6898cd2916f9 amazingly describing the steps to setup the container.
Just wanted to share as it might be useful for others.

**Comment by steveswinsburg** [comment]
When new Oracle database docker containers are released, I write up some simplified instructions for them, like this:
https://github.com/steveswinsburg/oracle21c-docker
https://github.com/steveswinsburg/oracle19c-docker
https://github.com/steveswinsburg/oracle12c-docker (modified container with some memory mods)
https://github.com/steveswinsburg/oracle12c-vagrant (does anyone even use Vagrant any more?!)

I find my development team likes these instructions rather than the more comprehensive ones as it just get to the point. Hope it helps someone also.
